rusk/src/lib/builder/http_only.rs

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use tokio::sync::broadcast;
+use tracing::info;
+
+use crate::http::{DataSources, HttpServer, HttpServerConfig};
+
+#[derive(Default)]
+pub struct RuskHttpBuilder {
+    http: Option<HttpServerConfig>,
+}
+
+impl RuskHttpBuilder {
```

Comment:
For the future, it would be probably a good idea to decouple building the `RuskHttp` service from running it and perhaps give the option to run some logic at `shutdown` too.

Reply:
Opened #2453 to address this